content/docs/studio/troubleshooting.md

```diff
@@ -78,24 +75,24 @@ import the repo again.
 - Failed to start parsing
 - Parsing stopped unexpectedly
 
-## Errors related to DVC remotes
+## Errors related to DVC remotes and credentials
 
-Iterative Studio can access data from network-accessible remotes such as Amazon
-S3, Microsoft Azure, etc but not from [local DVC remotes][local-dvc-remotes]. If
-your project uses an unsupported remote, you will see one of the following
-errors:
+Iterative Studio can include data from
+[data remotes](/doc/studio/user-guide/projects-and-experiments/configure-a-project#data-remotes-cloudremote-storage)
+in your project. However, it can access data from network-accessible remotes
+such as Amazon S3, Microsoft Azure, etc but not from [local DVC
+remotes][local-dvc-remotes]. If your project uses an unsupported remote, you
+will see one of the following errors:
 
 - Local remote was ignored
 - Remote not supported
 
 Please use one of the following types of data remotes: Amazon S3, Microsoft
 Azure, Google Drive, Google Cloud Storage and SSH.
 
-## Errors related to credentials
-
-If your repository uses data remotes that Iterative Studio needs to access, then
-you should [add the required credentials to your project][cloud-credentials]. If
-credentials are missing or incorrect, you will see one of the following errors:
+If the data remotes have access control, then you should [add the required
+credentials to your project][cloud-credentials]. If credentials are missing or
+incorrect, you will see one of the following errors:
```

Comment:
Why the changes here? The old wording makes more sense to me.

Reply:
To manage the total number of items at the top on the troubleshooting page, I merged the sections `Errors related to DVC remotes` and `Errors related to credentials` into a single section `Errors related to DVC remotes and credentials`. I did the rewording to make the content within this single section flow better from one paragraph to next (the previous paragraph already mentions `Iterative Studio can include data from data remotes..`).